src/git.rs

```diff
@@ -28,26 +27,24 @@ impl Git {
     pub fn commits(&self) -> Result<Vec<Commit>, Box<dyn Error>> {
         let git_log = self.git_log_cmd.get_log()?;
 
```

Comment:
Plus woul be great to have some comment in the code to explain why it is done this way.

Reply:
Yes, when compiled to wasm `Regex::new` would cause `npm` to exit with *success* code so even wrong changelog would be recognized as valid. I have no idea why it happens and didn't feel like investigating especially because it may be fixed in newer `regex` versions but I want to support older ones because of MSRV.